src/VisualStudio/Core/Def/Implementation/LanguageClient/InProcLanguageServer.cs

```diff
@@ -184,7 +183,7 @@ private async void DiagnosticService_DiagnosticsUpdated(object sender, Diagnosti
                     }
 
                     // Only publish document diagnostics for the languages this provider supports.
-                    if (document.Project.Language != LanguageNames.CSharp && document.Project.Language != LanguageNames.VisualBasic)
+                    if (document.Project.Language != CodeAnalysis.LanguageNames.CSharp && document.Project.Language != CodeAnalysis.LanguageNames.VisualBasic)
```

Comment:
why were these expanded?

Reply:
There are name conflicts in new types I use in this file when using Microsoft.CodeAnalysis and the LSP client packages.  Since the code in here is mainly LSP related, I opted for removing the import and expanding the CodeAnalysis types.